src/common/common.test.ts

```diff
@@ -529,27 +531,30 @@ describe("getVerifiableCredential", () => {
     let mockedFetch: jest.MockedFunction<typeof fetch>;
 
     beforeEach(() => {
-      mockedFetch = jest.fn<typeof fetch>().mockResolvedValueOnce(
-        new Response(undefined, {
-          status: 401,
-          statusText: "Unauthenticated",
-        }),
+      mockedFetch = mockedFetchWithResponse(
+        401,
+        `{"status": 401, "title": "Unauthorized", "detail": "Example detail"}`,
+        { "Content-Type": "application/problem+json" },
```

Comment:
Would it be helpful to expose a mock function from the base error library? I reckon we'll do this type of mock for all libraries being integrated.

Reply:
Perhaps, although there are probably other common mock functions that could be in a base library but not errors.